packages/gatsby-image/src/index.js

```diff
@@ -341,6 +342,7 @@ class Image extends React.Component {
           className={`${className ? className : ``} gatsby-image-wrapper`}
           style={divStyle}
           ref={this.handleRef}
+          key={this.props.tagKey || `fixed-${JSON.stringify(image.srcSet)}`}
```

Comment:
One nit: I might recommend this is added at the destructuring props level, i.e. [here](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-image/src/index.js#L194-L205)

Nice way of not duplicating, and it's where the other props are declared too!